lms/djangoapps/experiments/utils.py

```diff
@@ -5,11 +5,33 @@
 import json
 from student.models import CourseEnrollment
 from django_comment_common.models import Role
+from openedx.core.djangoapps.waffle_utils import WaffleFlag, WaffleFlagNamespace
 from course_modes.models import get_cosmetic_verified_display_price
 from courseware.access import has_staff_access_to_preview_mode
 from courseware.date_summary import verified_upgrade_deadline_link, verified_upgrade_link_is_valid
 from xmodule.partitions.partitions_service import get_user_partition_groups, get_all_partitions_for_course
 from opaque_keys.edx.keys import CourseKey
+from openedx.core.djangoapps.catalog.utils import get_programs
+
+
+# TODO: clean up as part of REVEM-199 (START)
+# .. feature_toggle_name: experiments.add_programs
+# .. feature_toggle_type: flag
+# .. feature_toggle_default: False
+# .. feature_toggle_description: Toggle for adding the current course's program information to user metadata
+# .. feature_toggle_category: experiments
+# .. feature_toggle_use_cases: monitored_rollout
+# .. feature_toggle_creation_date: 2019-2-25
+# .. feature_toggle_expiration_date: None
+# .. feature_toggle_warnings: None
+# .. feature_toggle_tickets: REVEM-63, REVEM-198
+# .. feature_toggle_status: supported
+PROGRAM_INFO_FLAG = WaffleFlag(
+    waffle_namespace=WaffleFlagNamespace(name=u'experiments'),
+    flag_name=u'add_programs',
+    flag_undefined_default=False
```

Comment:
Can you explain why you added the "flag_undefined_default" parameter here? I haven't seen that before.

Reply:
I mimic'ed our FBE switch. Happy to change this if we have a preference for something else

https://github.com/edx/edx-platform/blob/b034f4e0d42259af2e98758d792f436de9b8e1eb/openedx/features/course_duration_limits/config.py#L23

Reply:
Nah, I think you're good. I just wanted to know what it means.